.github/workflows/go.yml

```diff
@@ -0,0 +1,25 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ "dev", "master" ]
+  pull_request:
+    branches: [ "dev", "master" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+
+    - name: Build
+      run: go build -v
+
+    - name: Test
+      run: cd controllers && go test
```

Comment:
i think we should test all the tests in service, utils, controllers...

Reply:
When i run `go test` in root i get

<img width="468" alt="Screenshot 2023-01-02 at 9 08 43 PM" src="https://user-images.githubusercontent.com/72073401/210252771-665d5f09-241b-4d71-a7a2-3f2b918f91d0.png">

Any idea what modifications are require to run all the unit test present in the project by firing this cmd from the root

Reply:
check this 
https://stackoverflow.com/a/16353449

Reply:
Thanks!!